src/main/java/com/exactpro/remotehand/rhdata/RhScriptResult.java

```diff
@@ -28,10 +29,10 @@
 {
 	private int code = SUCCESS.getCode();
 	private String errorMessage;
-	private List<String> textOutput;
-	private List<String> screenshotIds;
-	private List<String> encodedOutput;
-	
+	private List<ActionResult> actionResults;
```

Comment:
Probably extending class ActionResult with screenshotId and endcodedOutput will better solution?